.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us find and fix the bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Affects version**
+State which version of gem5 this bug was found in. If on the develop branch state the Commit revision ID you are working.
+
+**gem5 Modifications**
+If you have modified gem5 in some way, please state, to the best of your ability, how it has been modified.
+
+**To Reproduce**
+Steps to reproduce the behavior. Please assuming starting from a clean repository:
+1. Compile gem5 with command ...
+2. Execute the simulation with...
```

Comment:
I would add something here about including the python script as well as the parameters that was used.

Reply:
Done